backend/src/timeoff/timeoff.model.ts

```diff
@@ -0,0 +1,23 @@
+import * as mongoose from 'mongoose';
+
+
+
+export const TimeoffSchema = new mongoose.Schema({
+    reason: { type: String, require: true },
+    from: { type: Date, require: true },
+    to: { type: Date, require: true },
+    by: { type: String, require: true },
+    status: { type: String, require: true, default:"Pendding" }, //Pendding, approve, reject
+    censor: { type: String, default: "" }
```

Comment:
what is censor? Is that means: Approver? Ithink we can use approver or PIC is ok